components/tools/OmeroPy/test/integration/gatewaytest/test_get_objects.py

```diff
@@ -571,6 +571,23 @@ def testGetImage(self, gatewaywrapper, author_testimg_tiny):
         assert image.getOwnerOmeName == testImage.getOwnerOmeName
         assert image.getThumbVersion() is not None
 
+    def testGetImageLoadPixels(self, gatewaywrapper, author_testimg_tiny):
+        testImage = author_testimg_tiny
+        conn = gatewaywrapper.gateway
+        # By default, don't load pixels
+        image = conn.getObject("Image", testImage.id)
+        assert not image._obj.pixelsLoaded
+        # Load just the pixels
+        image = conn.getObject("Image", testImage.id,
+                               opts={'load_pixels': True})
+        assert image._obj.pixelsLoaded
+        assert not image._obj._pixelsSeq[0].channelsLoaded
+        # Load pixels and channels
+        image = conn.getObject("Image", testImage.id,
+                               opts={'load_channels': True})
+        assert image._obj.pixelsLoaded
+        assert image._obj._pixelsSeq[0].channelsLoaded
```

Comment:
I have added a ```assert lc.getPhotometricInterpretation().isLoaded()``` but can't check illumination, mode or contrastMethod since they are not set for this test image.